doc/principe.md

```diff
@@ -0,0 +1,38 @@
+
+# Principe de fonctionnement du validateur de document
+
+## 1) Chargement du modèle de document
+
+Le modèle de document est chargé en fonction des options `--config/--version`.
+
+## 2) Préparation d'un dossier de validation
+
+Le validateur prépare un répertoire de travail pour la validation où il écrira :
+
+* Le rapport de validation (`validation/validation.jsonl`)
```

Comment:
typo : one extra character in file extension

Reply:
No, jsonl stands for [JSON Lines](http://jsonlines.org/)